services/typescript-service.ts

```diff
@@ -169,6 +181,9 @@ export class TypeScriptService implements ITypeScriptService {
 			.concat(this.getTypeScriptCompilerOptionsAsArguments(options.compilerOptions) || [])
 			.value();
 
+		// HACK: if we're using the project's own typescript module, do not pass any additional arguments
```

Comment:
The aim of this is that in the future we'll try and not pass any arguments other than the `cwd`.
So this is temporary until we get around to that